lib/capq/binheap_test.c

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright 2017 Kjell Winblad (http://winsh.me, kjellwinblad@gmail.com)
+ *
+ *  This file is part of kpqueue.
+ *
+ *  kpqueue is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  kpqueue is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with kpqueue.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdio.h>
+#include "binheap.h"
+
+int main()
```

Comment:
I don't think it hurts to keep it where it is but I can remove it or move it if you insist?